cjs/src/api/controllers/app.controller.js

```diff
@@ -1,15 +1,13 @@
-const express = require('express');
```

Comment:
@samuko-things 
You probably did not read the issue title and body well.
You should only be fixing the ts template (not the esm and cjs templates).

Delete this branch and close this PR. 

Create a new branch from the develop branch and start again.

Please read the issue again properly this time and let me know if you have any questions on the community channel.